soap/wsdl-loader.js

```diff
@@ -0,0 +1,103 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: generator-loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+var url = require('url');
+var chalk = require('chalk');
+
+var fs = require('fs');
+var async = require('async');
+var soapGenerator = require('loopback-soap');
+var soap = require('strong-soap').soap;
+var WSDL = soap.WSDL;
+var path = require('path');
+
+// loads remote WSDL or local WSDL using strong-soap module APIs.
+function loadWsdl(wsdlUrl, log, cb) {
+  WSDL.open(wsdlUrl, {}, function(err, wsdl) {
+    if (err) {
+      return cb(err);
+    }
+    console.log('loadWsdl loadWsdl %j', wsdl);
```

Comment:
@raymondfeng I have already removed console.log already in my local branch yesterday after the CI passed. I had add those just to debug statements to the PR to see what's going on in CI build on Travis, ubuntu + node4 environment since we don't have these environments setup locally.